Specs/DateTime/French/DateTimeModel.json

```diff
@@ -3206,6 +3206,7 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
+    "NotSupported": "java, python",
```

Comment:
When adding a commit message like this, please specify in which platforms you're disabling them.